src/app/globals.css

```
@@ -436,46 +406,19 @@
-  }
-
-  .reward-surface:hover {
-    transform: translateY(-3px);
-    border-color: rgba(255, 138, 102, 0.32);
-    box-shadow:
-      0 28px 70px rgba(0, 0, 0, 0.34),
-      0 0 0 1px rgba(255, 138, 102, 0.08),
-      inset 0 1px 0 rgba(255, 255, 255, 0.06);
+  .paper-inset {
+    @apply text-primaryText border;
+    background: theme("colors.dark.primaryBackground");
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)